### ISSUE TITLE
[Bug] Focus order wrong in Details view for add-ons

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#9793]
## Summary
At the bottom of the details view for a Fenix add-on (ublock origin in this particular case) there's a section that lists the author, version, last updated, homepage, and rating for the extension. It's a grid with two columns and the aforementioned 5 rows. The left column is the label like "Authors" and the right column is the value like "Raymond Hill". When I swipe through that grid the order is wrong. Talkback reads "Authors. Version. Raymond Hill. 1.26.0. Last updated. April 6, 2020...." 

For the first two rows the focus order is wrong. For the rest of the entries it's OK. 

## Steps to reproduce
Enable Talkback
Navigate to uBlock Origin Details view
Swipe through the page and note the wrong ordering

### Expected behavior
When swiping, focus and announcement should move from "Authors" to "Raymond Hill" to "Version" to "1.26.0" and so on.

### Actual behavior
When swiping, focus and announcement move from "Authors" to "Version" to "Raymond Hill" to "1.26.0". 

### Device information

* Android device: Galaxy s10e
* Fenix version: Sunday 4/5 @ 6:10 PM

**Comment by asadotzler** [comment]
This issue is an accessibility P2. The feature is available but difficult to use because of incorrect focus order. This shouldn't block the feature but we cannot call it a delightful experience without fixing this.

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on my Nightly debug **`Build #21040820 39.0.20200410130104, 464e3038d
GV: 77.0a1-20200409095500 AS: 0.55.0 from Monday 4/13`** using the following devices:
• Google Pixel 3a (Android 10) [`Talkback`]
• Samsung Gaaxy S7 (Android 7) [`Voice assistant`]

► Video
![20200413-111147](https://user-images.githubusercontent.com/51314259/79105031-82123400-7d78-11ea-84bd-9ddb38abf93f.gif)
